src/components/Card.js

```diff
@@ -15,9 +30,18 @@ export class Card {
   }
 
   getView() {
+    // console.log(this._ownerId, this._owner, this._cardId, this._cardLikes);
     this._cardElement = this._getElement();
     this._setAttributes();
     this._setEventListeners();
+    if (this._ownerId !== "b467d8eeb238431481a2ba2b") {
```

Comment:
`b467d8eeb238431481a2ba2b` 

This is a hardcode

`_id` of the user should be received from the server only 1 time. You need to store it in a variable `userId` or in class `UserInfo` and use it when you create a card passing it to the constructor of `Card`